docs/running-locally.md

```diff
@@ -154,7 +154,7 @@ To test SSO integration, use `PROFILE=sso`:
 make start UI=true PROFILE=sso
 ```
 
-## TLS
+### TLS
```

Comment:
This was a small oversight from https://github.com/argoproj/argo-workflows/pull/13674